src/test/java/com/github/dockerjava/core/command/ExecStartCmdImplTest.java

```diff
@@ -53,7 +51,7 @@ public void execStart() throws Exception {
         dockerClient.startContainerCmd(container.getId()).exec();
 
         ExecCreateCmdResponse execCreateCmdResponse = dockerClient.execCreateCmd(container.getId())
-                .withAttachStdout(true).withCmd("touch", "/execStartTest.log").exec();
+                .withAttachStdout(true).withCmd("touch", "/execStartTest.log").withUser("root").exec();
```

Comment:
Added `User` here to show that not only creating the command works but that executing with the user does as well.